docs/proposals/pepxxx_wheel_variant_support.md

```diff
@@ -762,9 +762,9 @@ returned by the variant provider plugin. Similarly, the order of property values
 `default-priorities.property`, with missing properties appended in order from the variant provider output. In each list,
 earlier elements have higher priority than later elements.
 
-A variant wheel has a higher priority than another variant wheel if its most important property is more important than
-the most important property of the other variant wheel. If both wheels have the same most important property, compare
-the second most important property, and so on, until a tie-breaker is found. If the same set of the most important
+A variant wheel has a higher priority than another variant wheel if its highest priority property is of higher priority
+than the highest priority property of the other variant wheel. If both wheels have the same highest priority property,
+compare the second highest priority property, and so on, until a tie-breaker is found. If the same set of highest priority
```

Comment:
Not really your fault and you don't need to change it right now, but I really wish we wouldn't be saying stuff like "the highest priority X of Y is of higher priority than the highest priority X of Z". This is a lot of "higher" and "highest" priorities in a single sentence, and that makes my brain melt.